hw/top_earlgrey/doc/_index.md

```diff
@@ -291,10 +243,8 @@ See the [pinmux specification]({{< relref "hw/ip/pinmux/doc" >}}) for how to con
 
 The chip contains one UART peripheral that implement single-lane duplex UART functionality.
 The outputs and inputs can be configured to any chip IO via the pinmux.
```

Comment:
might be worth adding a link to https://docs.opentitan.org/hw/ip/pinmux/doc/

Reply:
Done.